lib/utils/bgpstream_utils_rpki.h

```diff
@@ -30,11 +30,10 @@
 #ifndef __BGPSTREAM_UTILS_RPKI_H
 #define __BGPSTREAM_UTILS_RPKI_H
 
-#ifdef WITH_RPKI
-#include "bgpstream_elem_int.h"
+#include <stdint.h>
 #include <inttypes.h>
 #include <roafetchlib/roafetchlib.h>
-#include <stdint.h>
+#include "bgpstream_elem_int.h"
```

Comment:
This header is installed, right? We can't include an internal header from a publicly installed header.

Reply:
What do you need from the elem_int header?

Reply:
That was a typo, it should be ```bgpstream_elem.h```. My bad. 

Is the inclusion of ```bgpstream_elem.h``` okay or are you concerned about the direction of the inclusion?

Reply:
So long as it works, it is ok with me.

Reply:
okay great, it does (fixed).